modules/ROOT/pages/6.8-release-notes.adoc

```diff
@@ -121,6 +121,17 @@ The {productname} 6.8 release includes an accompanying release of the **AI Assis
 
 ==== Add selection-based option to ai shortcuts to restrict shortcut misuse
 //#TINY-10159
+In **AI Assistant 1.2.0**, the new selection property is available for shortcuts with prompts in the `ai_shortcuts` option. This `boolean` property controls when that shortcut is enabled based on the current selection within the editor. 
+
+For a shortcut with a prompt, when the property is:
+
+* **`true`**: this shortcut will only be enabled when user has made a selection in the editor;
+* **`false`**: this shortcut will only be enabled when the user has not made a selection in the editor;
+* **`undefined`**: or not set, this shortcut will always be enabled.
+
+This property allows the integrator to define shortcuts with `selection: true` which should only operate on context, and when the user has selected content. Alternatively, shortcuts with `selection: false` will only be used with no selection, allowing the integrator to define shortcuts for generating content from a specific prompt, without any context.
+
+As part of this update, all of the default shortcuts available in **AI Assistant 1.2.0** now have the property `selection: false`, as they are all shortcuts which require a selection context to generate content correctly.
```

Comment:
```suggestion
In **AI Assistant 1.2.0**, the new selection property is available for shortcuts with prompts in the xref:ai.adoc#ai_shortcuts[`+ai_shortcuts+` option]. This `+boolean+` property controls when that shortcut is enabled based on the current selection within the editor. 

For a shortcut with a prompt, when the property is:

* `+true+`: this shortcut will _only_ be enabled when user _has_ made a selection in the editor;
* `+false+`: this shortcut will _only_ be enabled when the user _has not_ made a selection in the editor;
* `+undefined+`: or _not set_, this shortcut will _always_ be enabled.

This property allows the definition of shortcuts which should only operate when the user has selected content, requiring the selection as context to the AI when the property is `+true+`. Additionally, shortcuts which are intended to generate specific content will not be enabled with any selection when the property if `+false+`.

As part of this update, all of the default shortcuts available in **AI Assistant 1.2.0** now have the property `+selection: false+`, as they are all shortcuts which require a selection context to generate content correctly.
```
Fixes for italics and styles and from https://github.com/tinymce/tinymce-docs/pull/2997